node/src/actors/chain_manager/handlers.rs

```diff
@@ -1309,26 +1309,26 @@ impl Handler<GetReputation> for ChainManager {
         };
 
         let identities = if all {
-            rep_eng.trs().identities().map(|(k, _v)| k).collect()
+            rep_eng.get_rep_ordered_ars_list()
         } else {
-            vec![&pkh]
+            vec![pkh]
         };
 
         let total_reputation = rep_eng.total_active_reputation();
         let reputation_hm = identities
             .into_iter()
             .map(|pkh| {
-                let reputation = rep_eng.trs().get(pkh);
-                let eligibility = rep_eng.get_eligibility(pkh) + 1;
-                let is_active = rep_eng.ars().contains(pkh);
+                let reputation = rep_eng.trs().get(&pkh);
+                let eligibility = rep_eng.get_eligibility(&pkh) + 1;
+                let is_active = rep_eng.ars().contains(&pkh);
```

Comment:
Wait, this is wrong. `is_active` will always be `true` now. What about identities with reputation that are not active?